src/components/follow.tsx

```diff
@@ -25,6 +25,28 @@ const styles = (kicker: string, inverted: string): SerializedStyles => css`
 	border: none;
 	background: none;
 	margin-left: 0;
+	margin-top: 4px;
+
+	svg {
```

Comment:
Perhaps worth considering a `css` attribute on the `<svg>` over type selectors? Not a blocker.